hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleManager.java

```diff
@@ -45,10 +50,9 @@ public class ThrottleManager {
     private static final ThrottleDefinitions DEFAULT_THROTTLE_DEFINITIONS = ThrottleDefinitions.DEFAULT;
     public static final Set<HederaFunctionality> expectedOps = ExpectedCustomThrottles.ACTIVE_OPS.stream()
             .map(protoOp -> HederaFunctionality.fromProtobufOrdinal(protoOp.getNumber()))
-            .collect(Collectors.toSet());
+            .collect(Collectors.toCollection(() -> EnumSet.noneOf(HederaFunctionality.class)));
```

Comment:
Just a question.  How does providing the empty `EnumSet` change semantics from the `toSet` collector?

Reply:
`expectedOps` is a `HashSet` when using `Collectors.toSet()` and an `EnumSet` with the new code.